tests/unit/src/test/scala/org/scalafmt/tests/InfixSuite.scala

```diff
@@ -31,4 +31,5 @@ object InfixSuite extends BaseScalaPrinterTest {
   check("a op[T] b")
   check("a b ((c, d))")
   check("a op (a: _*)")
+  check("new {} OP S")
```

Comment:
What about `new {}` or is the bug only related to infix operators?

Reply:
`new {} OP S` becomes `new OP S` There is probably other patterns that can cause this issue, but it's fixed at the root.

Reply:
Does `new {}` test this fix as well? I'm wondering if we can minimize the test case.

Reply:
yup, it's going to give a parse error!